experiments/AMIP/components/atmosphere/climaatmos.jl

```diff
@@ -21,6 +21,18 @@ struct ClimaAtmosSimulation{P, Y, D, I} <: Interfacer.AtmosModelSimulation
 end
 Interfacer.name(::ClimaAtmosSimulation) = "ClimaAtmosSimulation"
 
+hasradiation = function (integrator)
+    return !isnothing(integrator.p.atmos.radiation_mode)
+end
+
+hasmoisture = function (integrator)
+    return !(integrator.p.atmos.moisture_model isa CA.DryModel)
+end
+
+zero_scalar_half(integrator) = zeros(axes(CC.Fields.level(integrator.p.scratch.ᶠtemp_scalar, CC.Utilities.half)))
```

Comment:
Could you add comments explaining each of these? e.g. `_half` will return the first center (I think), `_one` will return the first face, and `_center` is the whole center space